.github/workflows/label_pr.yml

```diff
@@ -0,0 +1,34 @@
+on:
+  repository_dispatch:
+    types:
+      - label_pr
+
+env:
+  ACCESS_TOKEN: ${{ secrets.ACCESS_TOKEN }}
```

Comment:
Could we use `github.token` for this (or `GITHUB_TOKEN`) instead? Or does that not have enough permissions? I think it should :thinking: Would be nice to get rid of a secret to manage is all.

Reply:
We should make a separate issue to remove the `ACCESS_TOKEN` from all the workflows it's used it and use `GITHUB_TOKEN` in all those places.

Reply:
Created #182.